build.cake

```diff
@@ -73,12 +77,18 @@ Task("ReplaceStubWithExample")
 Task("TestUsingExampleImplementation")
     .IsDependentOn("ReplaceStubWithExample")
     .Does(() => {
-        var allProjects = GetFiles(buildDir + "/*/*.csproj");
+        var allProjects = GetFiles(exercisesBuildDir + "/*/*.csproj");
         Parallel.ForEach(allProjects, parallelOptions, (project) => DotNetCoreTest(project.FullPath));
     });
 
+Task("BuildGeneratorsSolution")
```

Comment:
Could you change this to `BuildGenerators`? The solution part is just an implementation detail.